tests/baselines/reference/user/bluebird.log

```diff
@@ -131,60 +131,62 @@ node_modules/bluebird/js/release/promise.js(65,15): error TS2350: Only a void fu
 node_modules/bluebird/js/release/promise.js(65,68): error TS2339: Property 'classString' does not exist on type 'typeof ret'.
 node_modules/bluebird/js/release/promise.js(95,22): error TS2339: Property 'isObject' does not exist on type 'typeof ret'.
 node_modules/bluebird/js/release/promise.js(99,59): error TS2339: Property 'classString' does not exist on type 'typeof ret'.
-node_modules/bluebird/js/release/promise.js(119,22): error TS2339: Property 'classString' does not exist on type 'typeof ret'.
-node_modules/bluebird/js/release/promise.js(121,32): error TS2339: Property 'classString' does not exist on type 'typeof ret'.
```

Comment:
"does not exist on type 'typeof ret'." is a correct error, but a very bad one. It happens because a module.exports a reference to itself, which causes `module.exports` inference to get confused and ignore the aliased variable's initialiser. `ret` contains a reference to its containing module by mistake, because bluebird has a globalThis shim that incorrectly includes `this`, even though when run inside a module, `this` is the module, not `globalThis`.